tests/e2e/solclient/solclient.go

```diff
@@ -157,20 +153,6 @@ func NewClient(cfg *NetworkConfig) (*Client, error) {
 	}, nil
 }
 
-func (c *Client) initSharedState() {
-	c.Accounts = &Accounts{
-		OCR:           solana.NewWallet(),
-		Store:         solana.NewWallet(),
-		Feed:          solana.NewWallet(),
-		Proposal:      solana.NewWallet(),
-		Owner:         solana.NewWallet(),
-		Mint:          solana.NewWallet(),
-		MintAuthority: solana.NewWallet(),
-		OCRVault:      solana.NewWallet(),
-		Authorities:   make(map[string]*Authority),
-	}
-}
```

Comment:
Please let me know where I can find OCR, Owner, and Feed in the new changes

Reply:
In ContractDeployer, each instance has all the accounts and authorities.